docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,264 @@
+
+The total coil temperature derate is calculated by :math:`\text{Coil Temperature Derate} = \frac{\text{otlo - temp}}{\text{otlo - otw}}` where the solution is saturated to [0,1].
+
+This means that for any temperature below otw, there is no derate applied. For any temperature above otlo, the microcontroller 
+temperature derate is 0. Any microcontroller temperature in that range is subject to a variable derate [0, 1].
+
+Suppose your module's otw is 140°C and its otlo is 150°C. This means that there will be no derate limiting applied under 140°C, your module will stop spinning at and above 150°C, and will be derate limited in between.
+
+A graph illustrating the microcontroller temperature derate calculation is provided below:
+
+.. image:: ../_static/manual_images/safety/coil_temp_derate.png
+    :align: center
+    :scale: 50%
+
+Calculation of the Total Derate
+-----------------------------------
+The total applied derate is the direct multiplication of the speed, microcontroller, and coil derate values. So, then our drive voltage system becomes:
+
+.. image:: ../_static/manual_images/safety/full_derate_system.png
+    :align: center
+    :scale: 50%
+
+Power Safety
+=================
+Power safety is a general monitoring interface that immediately coasts your module if any measured parameters reach dangerous levels. In the event that 
+:ref:`standard derates <derates>` are insufficient, the power safety interface ensures safe module operation.
+
+Safety Parameters Handled by the Power Safety Client
+-------------------------------------------------------------
+The power safety client manages measurements regarding input power (voltage and current), internal reference power, motor current, microcontroller temperature, and coil temperature.
+
+The full listing of power safety parameters can be found in the :ref:`Power Safety Client table <power_safety>`. Each safety has a low end limit as well as a high end limit. 
+If the measured parameter drops below the low end or goes above the high end, a fault flag is raised, and can be viewed in the ``fault_now`` parameter. 
+The ``fault_now`` parameter is only set by hardware. All faults that have occurred since the module's start up can be viewed in the ``fault_ever`` parameter. 
+The value of ``fault_ever`` is generally set by the hardware and cleared by the user.
+
+The module's behavior when faults are detected is determined by the ``fault_latching`` parameter. If ``fault_latching`` is set to 1, then any high flags 
+in ``fault_ever`` place the module into safe mode. In safe mode, the module is set to coast, and it will not spin or apply any control voltage. The module remains in safe mode until the user clears ``fault_ever``. 
+When ``fault_latching`` is set to 0, the module remains in safe mode until all power safety parameters are back in their safe operating range, and ``fault_now`` returns to 0.
```

Comment:
When fault_latching is set to 0, the module can come in and out of safe mode at high frequency.  This may be damaging to the the motor or supply since it can cause large ripple currents.  Vertiq recommends using fault_latching 1 whenever possible.